plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/CloudSdkProvider.java

```diff
@@ -36,11 +36,16 @@
   private final IPreferenceStore preferences;
 
    public CloudSdkProvider(IPreferenceStore preferences) {
+     // todo drop the null check and use the no args constructor instead
```

Comment:
Can you file an issue for this? Or would it be too big of a change fixing it in this PR?